conf.yaml

```diff
@@ -1138,9 +1138,9 @@ contents:
                     path:   shared/versions/stack/{version}.asciidoc
               - title:      Enterprise Search PHP client
                 prefix:     php
-                current:    7.16
-                branches:   [ 7.16 ]
-                live:       [ 7.16 ]
+                current:    *stackcurrent
+                branches:   [ master, 8.3, 8.2, 8.1, 8.0, 7.17, 7.16 ]
```

Comment:
This repo still uses a `master` branch. We can update this to `{main: master}` once we switch over to `main`.